prisma/migrations/20240319234708_reference_measurements/migration.sql

```diff
@@ -67,7 +67,7 @@ VALUES
     (180692, 'antler point count', 0, 10000, NULL, NULL),
     (202423, 'age', 0, 10000, NULL, 'The number of years that the animal has been alive for'),
     (202423, 'offspring count', 0, 10000, NULL, NULL),
-    (202423, 'tail length', 0, 10000, 'centimeter', NULL);
+    (202423, 'tail length', 0, 10000, 'centimeter', NULL); --This entry says that all animals have tails. This needs to change
```

Comment:
Did this get updated in the following migration?
If so, we can drop the changes in this file.

Reply:
I dont think so. I can't see any evidence of it being changed and Im not sure why I never got round to it. I will change this to a more appropriate TSN and push up here? Although, I am not sure of this will cause a lot of issues as there may have been more changes to main since this was branched off? Any advice would be great please

Reply:
Ah roger, @LouisThedroux that change is to an old migration file, which shouldn't happen.

Reply:
@LouisThedroux You should be okay, the PR will notify you if you have any conflicting changes with main and the last migration to go into Critterbase was 2 months ago.